containers/src/Data/Graph.hs

```diff
@@ -246,9 +246,20 @@ flattenSCCs :: [SCC a] -> [a]
 flattenSCCs = concatMap flattenSCC
```

Comment:
Could we also add `flattenSCCs1 :: [SCC a] -> NonEmpty a`? It seems appropriate.

Reply:
Did you mean `flattenSCCs1 :: NonEmpty (SCC a) -> NonEmpty a`?

Reply:
Oh right, it doesn't work out. Scratch that.  
`NonEmpty (SCC a) -> NonEmpty a` could be added, but I doubt it will be useful because `flattenSCCs` is usually applied to the output of `stronglyConnComp`/`stronglyConnCompR`, which has to return `[SCC a]`.